core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -87,29 +79,25 @@ public Sep31Service(
       Sep10Config sep10Config,
       Sep31Config sep31Config,
       Sep31TransactionStore sep31TransactionStore,
-      Sep31DepositInfoGenerator sep31DepositInfoGenerator,
       Sep38QuoteStore sep38QuoteStore,
       ClientService clientService,
       AssetService assetService,
       RateIntegration rateIntegration,
-      EventService eventService,
-      CustodyService custodyService,
-      CustodyConfig custodyConfig) {
+      CustomerIntegration customerIntegration,
```

Comment:
we can remove the `customerIntegration` if it is not being used

Reply:
Good catch. I missed this while rebasing.